core/oslib/audiobackend_alsa.cpp

```diff
@@ -97,7 +110,7 @@ static void alsa_init()
 	rc=snd_pcm_hw_params_set_rate_near(handle, params, &val, &dir);
```

Comment:
Should the sample rate be configurable or is 44.100Hz needed for something?

Reply:
44.1 is needed